packages/odyssey-react/.storybook/decorators/withGlobalReset.tsx

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ComponentType } from 'react';
+
+function withGlobalReset(Story: ComponentType) {
+  return (
+    <>
+      <style>{require('!to-string-loader!css-loader!@okta/odyssey/src/scss/base/_reset.scss')}</style>
```

Comment:
nice! I checked out the built storybook instance from CI and it doesn't appear to work though? [See this example from radio](https://b732bda.ods.dev/sb/index.html?path=/story/components-radio--default) vs an [older build without reset changes](https://6133699.ods.dev/sb/index.html?path=/story/components-radio--default). The `fieldset` border [isn't being set to 0 correctly](https://github.com/okta/odyssey/blob/develop/packages/odyssey/src/scss/base/_reset.scss#L111)

Reply:
This is expected, currently, only the [TextArea Story is making use of withGlobalReset](https://github.com/okta/odyssey/pull/1056/files#diff-e36bd870a2e0a549b0963187bf89ca95c952b334f543d72f50b4a13c61b3d23cR41)

I'm going to implement it across the board as a next step. Just wanted to run this commit by you as a confidence check. Will get it in everywhere now!

Reply:
Got it thanks for clarifying